agents/lakeshore372/LS372_agent.py

```diff
@@ -990,23 +1071,25 @@ def make_parser(parser=None):
     pgroup.add_argument('--ip-address')
     pgroup.add_argument('--serial-number')
     pgroup.add_argument('--fake-data', type=int, default=0,
-                        help='Set non-zero to fake data, without hardware.')
+            help='Set non-zero to fake data, without hardware.')
     pgroup.add_argument('--dwell-time-delay', type=int, default=0,
-                        help="Amount of time, in seconds, to delay data\
-                              collection after switching channels. Note this\
-                              time should not include the change pause time,\
-                              which is automatically accounted for.\
-                              Will automatically be reduced to dwell_time - 1\
-                              second if it is set longer than a channel's dwell\
-                              time. This ensures at least one second of data\
-                              collection at the end of a scan.")
+            help="Amount of time, in seconds, to delay data\
+                    collection after switching channels. Note this\
+                    time should not include the change pause time,\
+                    which is automatically accounted for.\
+                    Will automatically be reduced to dwell_time - 1\
+                    second if it is set longer than a channel's dwell\
+                    time. This ensures at least one second of data\
+                    collection at the end of a scan.")
     pgroup.add_argument('--mode', type=str, default='acq',
-                        choices=['idle', 'init', 'acq'],
-                        help="Starting action for the Agent.")
+            choices=['idle', 'init', 'acq'],
+            help="Starting action for the Agent.")
     pgroup.add_argument('--sample-heater', type=bool, default=False,
-                        help='Record sample heater output during acquisition.')
+            help='Record sample heater output during acquisition.')
     pgroup.add_argument('--enable-control-chan', action='store_true',
-                        help='Enable reading of the control input each acq cycle')
+            help='Enable reading of the control input each acq cycle')
+    #config file
+    pgroup.add_argument('--configfile', type=str, help='Yaml file for initializing 372 settings')
```

Comment:
Happy to see the help string, this'll automatically be added to the arguments list on the docs page. But I think we should also add an example to the ocs-site-configs block on that docs page.